depmap/1.explore_depmap.ipynb

```diff
@@ -0,0 +1,634 @@
+       "Melanoma_Skin                           1\n",
+       "Melanoma_Eye                            1\n",
+       "ADRENAL_CORTEX                          1\n",
+       "Brain                                   1\n",
+       "Name: tissue, dtype: int64"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# How many tissues\n",
+    "depmap_cell_df.tissue.value_counts()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "f777d88e-094c-4043-a574-8be8b43ee274",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "256\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "Lung adenocarcinoma                         77\n",
+       "Lung small cell carcinoma                   75\n",
+       "Melanoma                                    72\n",
+       "Glioblastoma                                58\n",
+       "Colon adenocarcinoma                        48\n",
+       "Bladder carcinoma                           36\n",
+       "Neuroblastoma                               36\n",
+       "Lung squamous cell carcinoma                35\n",
+       "Ewing sarcoma                               33\n",
+       "Plasma cell myeloma                         31\n",
+       "Esophageal squamous cell carcinoma          28\n",
+       "Tongue squamous cell carcinoma              28\n",
+       "Pancreatic ductal adenocarcinoma            27\n",
+       "Adult acute myeloid leukemia                27\n",
+       "Breast carcinoma                            27\n",
+       "Renal cell carcinoma                        25\n",
+       "Clear cell renal cell carcinoma             25\n",
+       "Intrahepatic cholangiocarcinoma             25\n",
+       "Pancreatic adenocarcinoma                   23\n",
+       "High grade ovarian serous adenocarcinoma    23\n",
+       "Name: Cellosaurus_NCIt_disease, dtype: int64"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# What are the diseases\n",
+    "print(depmap_cell_df.Cellosaurus_NCIt_disease.nunique())\n",
+    "depmap_cell_df.Cellosaurus_NCIt_disease.value_counts().head(20)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "63d78068-f81a-4ee8-a519-bb76ec90c541",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Male      991\n",
+       "Female    747\n",
+       "Name: sex, dtype: int64"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# What are the sex distribution\n",
+    "depmap_cell_df.sex.value_counts()"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
It looks like this may have been run from outside the `drug-viability` environment.

Reply:
will fix